payload/sp/ItemCommand.cc

```diff
@@ -0,0 +1,112 @@
+#include "ItemCommand.hh"
+extern "C" {
+#include "ItemCommand.h"
+
+#include "sp/Commands.h"
+
+#include <game/item/ItemDirector.h>
+}
+#include <game/system/RaceConfig.hh>
+
+extern "C" {
+#include <revolution.h>
+}
+
+namespace SP {
+
+s8 GetMyPlayerID(void) {
+    auto *raceConfig = System::RaceConfig::Instance();
+    if (raceConfig == nullptr) {
+        OSReport("&cNot in a race.\n");
+        return -1;
+    }
+
+    auto &raceScenario = raceConfig->raceScenario();
```

Comment:
```suggestion
    auto &raceScenario = System::RaceConfig::Instance()->raceScenario();
```
RaceConfig is preserved through all scenes and is created on game start.

Reply:
This code really was busted before I rewrote it.

Reply:
I'll reimplement an actual check after #803 allows me to get the SceneManager.

Reply:
Actually, we can use the section ID to do this.